### DISCUSSION TITLE
How to use regex in API filters

### DISCUSSION THREAD

**Discussion by aabulmagd** [pocketbase/pocketbase#4139] [Q&A]
I want to query a collection to list records with field1 starts with certain letters. how can I achieve this?

**Comment by ganigeorgiev** [comment]
There is no support for regex when using the filters.

Even on SQLite level I don't think is doable at the moment because it requires loading a special user defined regex function which we don't do.

If you know the exact prefix your string is starting with, you ca apply a simple `LIKE` expression using the `~` operator:

```js
// will match any string starting with "abc" (case insensitive)
// (the `%` symbol matches any characters after the prefix)
yourField ~ "abc%"
```

**Reply by bobwatcherx** [reply]
can you give example url how to find data with a title that contains what I want to search for

https://poophdpocket.pockethost.io/api/collections/playerx/records?filter=title%3D%27DVDES%27

**Reply by bobwatcherx** [reply]
solved thx
